src/Cameras/camera.ts

```diff
@@ -404,6 +412,15 @@ export class Camera extends Node {
         return ret;
     }
 
+    /**
+     * Automatically tilts the projection plane, using `projectionPlaneTilt`, to correct the perspective effect on vertical lines.
+     */
+    public applyVerticalCorrection() {
+        const rot = this.absoluteRotation.toEulerAngles();
+
+        this.projectionPlaneTilt = this._scene.useRightHandedSystem ? -rot.x : rot.x;
```

Comment:
Just want to be sure - do we need to flip rotation.x on right handed systems?

Reply:
`toEulerAngles` adds a PI rotation about the Y axis in right handed systems, and since this rotation is applied before the X rotation, the X axis is flipped

Reply:
seems legit :-) but I stuggle so much with this I am always impressed by the ones who know... A bit like right and the other right...